docs/source/quick_start.rst

```diff
@@ -0,0 +1,53 @@
+.. _quick_start:
+
+Quick start
+===========
+
+Sample data
+-----------
+
+*xdem* comes with some sample data that is used throughout this documentation to demonstrate the features. If not done already, the sample data can be downloaded with the command
```

Comment:
When we talk about the package name we write "xDEM", and for the module sometimes ``xdem`` or *xdem*. Which one should we pick and stick to?
In the NumPy docs, looks like they stick to the package name "NumPy" without any kind of highlighting, and that the lower case, package-style naming ``numpy`` is only used with a submodule e.g. ``numpy.array``

Reply:
I agree. I have no strict opinion on what we should use. In case of NumPy, the name is somewhat highlighted with the camel syntax. It feels a bit weird to write xdem in the text as if it was a regular word, so I like having a way to highlight it.
We agreed that lowercase is better than uppercase so that's why I went with the italic (*xdem*) and I think the code highlight (`xdem`) should be reserved for actual code so not in the text.
What do you think?

Reply:
Just looked at what is done in the docs of Matplotlib, NumPy, SciPy, PyTorch, GPyTorch, I'd do the same: use the "normal text" package syntax that is camel-case/acronym-coded.
For us, it would correspond to using "xDEM" in normal text everywhere (in a way, its capitalization would serve to highlight the name).

Reply:
**Still time to change this later:** to avoid the discrepancy between the ``xdem`` and *xdem* in the docs of this PR, I changed into xDEM everywhere to mirror the syntax used in big packages as described above. It will be easier to replace if we want to, because with the capitalized letters it doesn't mix with the code ocurrences :)